railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -63,13 +63,15 @@ def configru
       template "config.ru"
     end
 
-    def gitignore
+    def gitfiles
       template "gitignore", ".gitignore"
+      template "gitattributes", ".gitattributes"
     end
 
     def version_control
       if !options[:skip_git] && !options[:pretend]
         run "git init", capture: options[:quiet]
+        run "git config diff.rails_credentials.textconv 'bin/rails credentials:show'", capture: options[:quiet]
```

Comment:
This only helps the first person to create the repository... maybe `bin/setup` would be better?

Reply:
+1 I think that `bin/setup` is a more appropriate place. Also, since, added it to `bin/setup`, I think that it is not necessary to run this with rails new.